diff --git a/Assets/AssemblyShadowDemo/Tests/Editor/AssemblyShadowDemo.EditorTests.asmdef b/Assets/AssemblyShadowDemo/Tests/Editor/AssemblyShadowDemo.EditorTests.asmdef
index 2a83686..52ac5c5 100644
--- a/Assets/AssemblyShadowDemo/Tests/Editor/AssemblyShadowDemo.EditorTests.asmdef
+++ b/Assets/AssemblyShadowDemo/Tests/Editor/AssemblyShadowDemo.EditorTests.asmdef
@@ -4,6 +4,7 @@
     "AssemblyShadowDemo.Editor",
     "AssemblyShadowDemo.Bootstrap",
     "HybridCLR.Editor",
+    "Unity.HybridCLR.AssemblyShadow.CodeGen",
     "HybridCLR.Runtime",
     "AssemblyA.Contracts",
     "AssemblyA.Implementation.Extensibility",
